rotkehlchen/db/upgrades/v45_v46.py

```diff
@@ -68,11 +69,7 @@ def _refresh_icons(write_cursor: 'DBCursor') -> None:
     @progress_step(description='Moving EVM event extra data to the history_events table.')
     def _move_extra_data(write_cursor: 'DBCursor') -> None:
         write_cursor.execute('ALTER TABLE history_events ADD COLUMN extra_data TEXT;')
-        write_cursor.execute(
-            'UPDATE history_events SET extra_data = '
-            '(SELECT extra_data FROM evm_events_info '
-            'WHERE evm_events_info.identifier = history_events.identifier);',
-        )
+        # no need to migrate the data since evm events are to be redecoded after this upgrade.
```

Comment:
I see why you would think this, but no you can't do this. Not all events are redecoded. All customized events are kept as they are. So at least those would need to be migrated. So you lose nothing by leaving this in as is, and removing it introduces bugs.